tutorial/README.md

```diff
@@ -0,0 +1,824 @@
+# OAM Tutorial
+
+This tutorial uses OAM with ARM-DSL to deploy an example microservices application with a Javascript Web UI, a Postgres database, and a set of REST API microservices.
+
+The application being deployed is shown in the following diagram:
+
+![Application architecture diagram](service-tracker-diagram.jpg)
+
+> Full application original source here:
+> https://github.com/chzbrgr71/service-tracker
+
+## Getting started
+
+Complete the getting started section in the root of this repository. You should be able to connect via `ssh` to the control plane VM as well as via `kubectl` to the AKS cluster.
```

Comment:
Also, mention that that the VM is the control panel for OAM and the AKS cluster is where the workloads will be deployed to. This difference is important. I found myself figuring this out by myself after reading this since it gave the impression that I could use OAM via VM *or* AKS directly, which is not the case.